packages/earth-admin/src/pages-client/layers/details.tsx

```diff
@@ -493,7 +492,12 @@ export function LayerDetail(props: any) {
                           control={control}
                           getOptionLabel={(option) => option.name}
                           getOptionValue={(option) => option.id}
-                          loadFunction={LayersService.getAllLayers}
+                          loadFunction={(query) =>
+                            LayersService.getAllLayers({
+                              ...query,
+                              filter: ['slug', '!=', slug].join(''),
```

Comment:
That's an interesting approach. Any reason not to do this?
```suggestion
                              filter: `slug!=${slug}`,
```